resources/components/ClientsForm.vue

```diff
@@ -45,12 +45,39 @@
         </div>
         <div class="col-6 col-sm-4 col-md-3 form-group">
             <label for="status">Status</label>
-            <select id="status" v-model="client.status" name="paid" type="select"
+            <select id="status" v-model="client.status" type="select"
                  class="form-control form-control-sm">
               <option value="unsettled">Nierozliczono</option>
-              <option value="settled">Rozliczono</option>
+                <option value="settled">Rozliczono</option>
             </select>
         </div>
+        <div class="col-12">
+            <div class="form-check-inline">
+                <input id="unregistered" v-model="client.unregistered" type="checkbox"
```

Comment:
W tym słowniku kolejna metoda z jakimiś labelami ? Gdzie robi jeżeli jest i to przekazać do vue ale nie czy czy warto i łatwo to zrobić 🤔